java/ql/src/semmle/code/java/frameworks/Assertions.qll

```diff
@@ -27,6 +28,13 @@ private predicate assertionMethod(Method m, AssertKind kind) {
     m.hasName("fail") and kind = AssertKindFail()
   )
   or
+  exists(RefType hamcrest |
+    m.getDeclaringType() = hamcrest and
+    hamcrest.hasQualifiedName("org.hamcrest", "MatcherAssert") // TODO: Check some older versions of hamcrest
```

Comment:
We typically try to avoid leaving TODO comments in `.ql` or `.qll` files. Could you address this prior to merging, if possible, or alternatively file a JIRA issue with a link to this line.

Reply:
Sure thing, I have now checked the older versions and they are the same. Comment removed.